### DISCUSSION TITLE
Option to set new tab page to homepage.

### DISCUSSION THREAD

**Discussion by aTosser** [zen-browser/desktop#2391] [Ideas]
Not Firefox Home. Or Zen Home if you wind up rebranding it.

**Comment by kylerm42** [comment]
Firefox used to have an option to set the new tab to any URL, but removed it a few years ago. Now we're reliant on extensions for this (I currently use [New Tab Override](https://addons.mozilla.org/firefox/addon/new-tab-override/)) but this would be great for the Zen team to bring back. Unfortunately, it seems they're going the opposite direction and getting rid of the new tab page completely.
